core/unit_test/CMakeLists.txt

```diff
@@ -158,6 +158,7 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;OpenMPTarget;OpenACC;HIP;SYCL)
         Other
         RangePolicy
         RangePolicyRequire
+        RangePolicyConstructors
```

Comment:
I'm not sure if this is the correct place in the `CMakeList.txt`, the layout is a but confusing to me.